nixos/maintainers/scripts/ec2/amazon-base-config.nix

```diff
@@ -1,5 +1,5 @@
 { modulesPath, ...}:
 {
-  imports = [ "${modulesPath}/virtualisation/amazon-config.nix" ];
+  imports = [ ../../../../nixos/modules/virtualisation/amazon-image.nix ];
```

Comment:
This file is copied to `/etc/nixos/configuration.nix`, so using relative paths is wrong. Please use `<nixpkgs/...>' instead.